openassessment/xblock/staff_area_mixin.py

```diff
@@ -236,8 +236,6 @@ def render_student_info(self, data, suffix=''):  # pylint: disable=W0613
         try:
             student_username = data.params.get('student_username', '')
             path, context = self.get_student_info_path_and_context(student_username)
-            expanded_view = data.params.get('expanded_view', [])
```

Comment:
Note to reviewers: I moved this functionality directly into the JavaScript instead of sending it to the server, setting classes in the HTML, and then having to deal with them in either CSS/JS.